app/experimenter/nimbus-ui/src/components/TableBranches/index.tsx

```diff
@@ -0,0 +1,91 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import {
+  getExperiment_experimentBySlug,
+  getExperiment_experimentBySlug_referenceBranch,
+  getExperiment_experimentBySlug_treatmentBranches,
+} from "../../types/getExperiment";
+import { Table } from "react-bootstrap";
+import NotSet from "../NotSet";
+
+type Branch =
+  | getExperiment_experimentBySlug_referenceBranch
+  | getExperiment_experimentBySlug_treatmentBranches;
+
+const TableBranches = ({
+  experiment,
+}: {
+  experiment: getExperiment_experimentBySlug;
+}) => {
+  const { featureConfig } = experiment;
+  const hasSchema = featureConfig !== null && featureConfig.schema !== null;
```

Comment:
nit: Should this be `featureConfig?.schema !== null` to avoid a second comparison?